src/interface/read_config_file.py

```diff
@@ -0,0 +1,108 @@
+import xml.etree.ElementTree as etree
+class ReadConfigFile():
+    
+    def __init__(self, fileconfig): 
+            
```

Comment:
no documentation for the parameter, and the parameter name is not following the standards